azure-pipelines/builds/ci.yml

```diff
@@ -2,15 +2,13 @@ trigger:
   batch: true
   branches:
     include:
-    - master
     - main
     - release/*
     - dev/source-build-5.0
```

Comment:
Is this still applicable?  I didn't find this branch.

Reply:
This might be a good opportunity to clean it up if it isn't needed anymore.

Reply:
Thanks.  Fixed.